### PR TITLE
Basic usage instructions.

### PR DESCRIPTION
**Pull request by dvdcastro**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,14 +20,14 @@ which bad pull requests can slip through. If that's not acceptable, please
 consider a more elaborate approach than this one. This is a proof-of-concept
 that might be good enough for some folks. Some edge cases are:
 
-- Pull requests get merged in quick succession:  
+- Pull requests get merged in quick succession:
```
Comment (twz123) [inline README.md:23]: Can you re-add the trailing whitespace? In Markdown, this is a line break.

**Review by twz123 — CHANGES_REQUESTED** [review]
Wow, thanks a lot. Just a little nit.

File: README.md (hunk 1 above)
Comment (dvdcastro) [inline README.md:23]:
```suggestion
- Pull requests get merged in quick succession:  
```

File: README.md [hunk 2]
```diff
@@ -20,14 +20,14 @@ which bad pull requests can slip through. If that's not acceptable, please
...
   Pull requests that were passing all of their status checks before the update
   of the target branch will remain in a mergeable state until the GitHub Action
   for the target branch triggered their respective builds. Usually, this time
   window is quite small, say, under a minute, but if the triggering GitHub
   action fails or gets stuck in the execution queue, this can go unnoticed for
   quite a while.
-- Stale pull requests:  
+- Stale pull requests:
```
Comment (dvdcastro) [inline README.md:30]:
```suggestion
- Stale pull requests:  
```

**Review by twz123 — APPROVED** [review]

**Comment by twz123** [comment]
Thanks!

**Pull request merged**